# Verify if there is one raised foot
string support_polygon
